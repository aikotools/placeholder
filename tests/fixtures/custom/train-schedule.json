{
  "scheduleId": "{{gen:uuid:schedule-001}}",
  "train": {
    "designation": "{{train:icenumber:723}}",
    "type": "{{train:type:ICE}}",
    "operator": "DB Fernverkehr"
  },
  "route": {
    "departure": {
      "station": "{{train:station:Berlin Hbf}}",
      "platform": "{{train:platform:7}}",
      "scheduledTime": "{{time:calc:0:HH\\:mm}}",
      "actualTime": "{{time:calc:300:HH\\:mm}}",
      "delay": "{{train:delay:5}}"
    },
    "arrival": {
      "station": "{{train:station:MÃ¼nchen Hbf}}",
      "platform": "{{train:platform:3}}",
      "scheduledTime": "{{time:calc:21600:HH\\:mm}}",
      "estimatedTime": "{{time:calc:21900:HH\\:mm}}",
      "delay": "{{train:delay:5}}"
    }
  },
  "metadata": {
    "createdAt": "{{time:calc:0:seconds}}",
    "validFrom": "{{time:calc:0:yyyy-MM-dd}}",
    "validUntil": "{{time:calc:604800:yyyy-MM-dd}}",
    "version": "{{gen:number:1}}"
  }
}
