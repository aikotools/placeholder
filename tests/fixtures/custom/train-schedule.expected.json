{
  "scheduleId": "schedule-001",
  "train": {
    "designation": "ICE 723",
    "type": "ICE",
    "operator": "DB Fernverkehr"
  },
  "route": {
    "departure": {
      "station": "Berlin Hbf",
      "platform": 7,
      "scheduledTime": "12:00",
      "actualTime": "12:05",
      "delay": 5
    },
    "arrival": {
      "station": "MÃ¼nchen Hbf",
      "platform": 3,
      "scheduledTime": "18:00",
      "estimatedTime": "18:05",
      "delay": 5
    }
  },
  "metadata": {
    "createdAt": 1742040000,
    "validFrom": "2025-03-15",
    "validUntil": "2025-03-22",
    "version": 1
  }
}
